docs/release_notes.md

```diff
@@ -4,14 +4,44 @@
 
 For a list of all changes in this release, see the [full changelog](https://github.com/nci/scores/compare/1.2.0...develop). Below are the changes we think users may wish to be aware of.
 
+### Introduced Support for Python 3.13 and Dropped Support for Python 3.9
+
+- Introduced support for Python 3.13 and dropped support for Python 3.9. See [PR #710](https://github.com/nci/scores/pull/710).
```

Comment:
Is there any need/benefit to add something along the lines of "`scores` now supports Python 3.10, 3.11, 3.12 and 3.13", or is it best left as is?

Reply:
I prefer it left as is, but happy to hear differently

Reply:
I left as is.